app/src/ui/lib/list/list.tsx

```diff
@@ -386,6 +386,8 @@ export class List extends React.Component<IListProps, IListState> {
         this.props.selectionMode !== 'single'
       ) {
         this.addSelection('down', event)
+      } else if (event.ctrlKey) {
```

Comment:
Is <kbd>Ctrl+Down</kbd> a common pattern for navigating lists on Windows or macOS? It's not a pattern I'm familiar with so I'd feel a lot more confident in merging this change if you could provide some links to platform-specific documentation or list some built-in apps that use this pattern. If it's just one of the platforms then we should conditionalize it to that platform.

<kbd>Ctrl+Up</kbd> and <kbd>Ctrl+Down</kbd> seems to be system shortcuts on macOS related to Window management.

![image](https://user-images.githubusercontent.com/634063/84511738-44258380-acc7-11ea-8449-5f3fa162137a.png)